### PR TITLE
Fix yarn postcss

### PR DESCRIPTION
**Pull request by maestromac**
## What type of PR is this? (check all applicable)
- [x] Bug fix

## Description
We use postcss to ensure proper browser compatibility for our CSS and it has been broken for a long time.

We first hid the output into a file because it was outputting a large amount of warning and sometime between then and now it stopped working completely.

I changed the extension it needed with the latest similar tool, remove redundant code and dealt with some edge cases

## Related Tickets & Documents
finally, resolves https://github.com/forem/forem/issues/14052

## QA Instructions, Screenshots, Recordings
test should pass

## Added tests?
- [x] no, because they aren't needed

## Added to documentation?
- [x] no documentation needed

## [optional] Are there any post deployment tasks we need to perform?
n/a

### CODE REVIEW

File: .github/workflows/ci-cd.yml [hunk 1]
```diff
@@ -128,6 +128,8 @@ jobs:
         uses: ruby/setup-ruby@v1
         with:
           bundler-cache: true
+      - run: bundle exec rails assets:precompile
+      - run: yarn run postcss
```
Comment (maestromac) [inline .github/workflows/ci-cd.yml:132]: Dedicated task so failure will be detected on CI.

**Review by Ridhwana — APPROVED** [review]
Tests pass and I pulled down the branch and just poked around the interface, things are working.

**Review by fdocr — APPROVED** [review]
